lib/cog_api/client.ex

```diff
@@ -83,4 +83,5 @@ defmodule CogApi.Client do
   @callback user_create(%Endpoint{}, %{}) :: {atom, %User{}}
   @callback user_update(%Endpoint{}, String.t, %{}) :: {atom, %User{}}
   @callback user_delete(%Endpoint{}, String.t) :: atom
+  @callback user_chat_handles(%Endpoint{}, String.t) :: {:ok, [%ChatHandle{}]}
```

Comment:
I know that a user is required for a chat handle `show` action but it seems like this belongs on the `ChatHandle` module instead.

I think a good principle (which I've only recently decided on) is that every function in a module should return the resource of the same name. `User` should only return `%User{}`. Thoughts?